runtimes/kestrel/src/lib.rs

```diff
@@ -676,9 +677,9 @@ impl pallet_multisig::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type RuntimeCall = RuntimeCall;
 	type Currency = Balances;
-	type DepositBase = constants::multisig::DepositBase;
-	type DepositFactor = constants::multisig::DepositFactor;
-	type MaxSignatories = constants::multisig::MaxSignitors;
+	type DepositBase = ConstU128<{ constants::multisig::DEPOSIT_BASE }>;
+	type DepositFactor = ConstU128<{ constants::multisig::DEPOSIT_FACTOR }>;
+	type MaxSignatories = ConstU32<{ constants::multisig::MAX_SIGNITORS }>;
```

Comment:
Yeah, I like this approach more that relies on well-known Rust primitives like `const`s and const generic, rather than some Substrate macro. What do you think?

Reply:
I reverted the changes in here: https://github.com/KILTprotocol/kilt-node/pull/797/commits/08e3236e09db6ea4974bb3729551d4ef1f106993. I will change all of them if you agree in a separate PR.

Reply:
I think being more sticky to rust primitives is better than substrate

Reply:
Will do!